src/classes/ACCT_IndividualAccounts_TEST.cls

```diff
@@ -955,13 +1079,453 @@ private class ACCT_IndividualAccounts_TEST {
+    */
+    @isTest
+    private static void sortContactsWithCustomFormat() {
+        UTIL_CustomSettingsFacade.getSettingsForTests(new Hierarchy_Settings__c
+                                                    (Account_Processor__c = UTIL_Describe_API.getHhAccRecTypeID(),
+                                                     Household_Account_Naming_Format__c = '{!{!FirstName} {!LastName}} House',
+                                                    Automatic_Household_Naming__c = true));
+
+		Contact con = UTIL_UnitTestData_API.getContact();
+        con.FirstName = 'Marvin';
+        con.LastName = 'Johnston';
+        insert con;
+
+        Contact requeryInsertedCon = [SELECT Id, AccountId, Account.Name
+                                      FROM Contact
+                                      WHERE Id = :con.Id];
+        Account a = [SELECT Id, Name
+                     FROM Account
+                     WHERE Id = :requeryInsertedCon.AccountId];
+
+        Contact con2 = new Contact(LastName = 'Johnston', FirstName = 'Tyrone', AccountId = a.Id);
+        insert con2;
+
+        Account requeryAccount = [SELECT Id, Name
+                                  FROM Account
+                                  WHERE Id = :requeryInsertedCon.AccountId];
+        System.assertEquals('Marvin Johnston and Tyrone Johnston House', requeryAccount.Name);
+
+        Contact con3 = new Contact(LastName = 'Phillips', FirstName = 'Crystal', AccountId = a.Id);
+        insert con3;
+
+        Account requeryFinalAccount = [SELECT Id, Name
+                                  	   FROM Account
+                                       WHERE Id = :requeryInsertedCon.AccountId];
+        System.assertEquals('Marvin Johnston, Tyrone Johnston and Crystal Phillips House', requeryFinalAccount.Name);
+    }
+
+    /*********************************************************************************************************
+    * @description Test Method to test sorting Contacts alphabetically with custom format.
+    * {!Salutation} {!{!FirstName}} {!LastName} Family
+    */
+    @isTest
+    private static void sortContactsWithCustomFormat2() {
+        UTIL_CustomSettingsFacade.getSettingsForTests(new Hierarchy_Settings__c
+                                                    (Account_Processor__c = UTIL_Describe_API.getHhAccRecTypeID(),
+                                                     Household_Account_Naming_Format__c = '{!Salutation} {!{!FirstName}} {!LastName} Family',
+                                                    Automatic_Household_Naming__c = true));
+
+		Contact con = UTIL_UnitTestData_API.getContact();
+        con.Salutation = 'Ms.';
+        con.FirstName = 'Betty';
+        con.LastName = 'White';
+        insert con;
+
+        Contact requeryInsertedCon = [SELECT Id, AccountId, Account.Name
+                                      FROM Contact
+                                      WHERE Id = :con.Id];
+        Account a = [SELECT Id, Name
```

Comment:
Minor, but we didn't actually need to query for the account here. Line 1514 below could simply be `Contact con2 = new Contact(LastName = 'White', FirstName = 'Lily', Salutation = 'Ms.', AccountId = requeryInsertedCon.AccountId);`